apps/etterlatte-saksbehandling-ui/client/src/shared/api/trygdetid.ts

```diff
@@ -105,12 +105,37 @@ export type IOpplysningsgrunnlag = {
   }
 }
 
+export interface IDetaljertBeregnetTrygdetid {
+  resultat: IDetaljertBeregnetTrygdetidResultat
+  tidspunkt: Date
+}
+
 export interface IDetaljertBeregnetTrygdetidResultat {
+  faktiskTrygdetidNorge: IFaktiskTrygdetid
```

Comment:
disse kan være null fra backend - vet ikke om det må merkes?

Reply:
Gjorde det nullable for å ha det tydelig. Ingen praktisk betydning.